src/main/java/uk/gov/digital/ho/proving/income/api/NinoUtils.java

```diff
@@ -1,20 +1,48 @@
 package uk.gov.digital.ho.proving.income.api;
 
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
 import java.util.regex.Pattern;
 
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+@Component
 public class NinoUtils {
```

Comment:
Stuart has pointed out that if want to create a shared library we'd have some work to do configuring an object repo, and a decision to make whether to go with the ACP or EBSA repo (or both).  And also this would create a barrier by requiring access to our internal object repo(s) in order to build the project.  So it needs some thought.

I suggest we leave it for now but I will try to generate a discussion around this because it feels like it won't be long before avoiding shared libraries will start to hurt us.

Reply:
Tech debt raised to agree on a strategy for shared code: https://jira.ipttools.info/browse/EE-3936